Documentation/devicetree/bindings/reserved-memory/linaro,secure-heap.yaml

```diff
@@ -0,0 +1,56 @@
+# SPDX-License-Identifier: (GPL-2.0 OR BSD-2-Clause)
+%YAML 1.2
+---
+$id: http://devicetree.org/schemas/reserved-memory/linaro,secure-heap.yaml#
+$schema: http://devicetree.org/meta-schemas/core.yaml#
+
+title: Linaro Secure DMABUF Heap
+
+maintainers:
+  - Olivier Masse <olivier.masse@nxp.com>
+
+description:
+  Linaro OP-TEE firmware need a reserved memory for the
```

Comment:
s/need/needs/

"needs a reserved memory" or "needs one or more reserved memories"?

Reply:
only one secure reserved memory is needed for OPTEE SDP.
however more could be created for other component, maybe the description should introduce that also ?

Reply:
the implementation is currently limiting the number of secure heap with the MAX_SECURE_HEAP macro.
which is currently set to 2.
It could be interesting to add this information to the documentation ?